docs/community/code-of-conduct.rst

```diff
@@ -0,0 +1,101 @@
+.. _code-of-conduct:
+
+=================
+ Code of Conduct
+=================
+
+We endorse the `Python Community Code of Conduct
+<https://www.python.org/psf/codeofconduct/>`_:
+
+   The Invenio community is made up of members from around the globe
+   with a diverse set of skills, personalities, and experiences.  It is
+   through these differences that our community experiences great
+   successes and continued growth.  When you're working with members of
+   the community, we encourage you to follow these guidelines which
+   help steer our interactions and strive to keep Invenio a positive,
+   successful, and growing community.
+
+   A member of the Invenio community is:
+
+   1. **Open.** Members of the community are open to collaboration,
+   whether it's on RFCs, patches, problems, or otherwise.  We're
+   receptive to constructive comment and criticism, as the experiences
+   and skill sets of other members contribute to the whole of our
+   efforts.  We're accepting of all who wish to take part in our
+   activities, fostering an environment where anyone can participate
+   and everyone can make a difference.
+
+   2. **Considerate.** Members of the community are considerate of
+   their peers -- other Invenio users.  We're thoughtful when
+   addressing the efforts of others, keeping in mind that often times
+   the labor was completed simply for the good of the community.
+   We're attentive in our communications, whether in person or online,
+   and we're tactful when approaching differing views.
+
+   3. **Respectful.** Members of the community are respectful.  We're
+   respectful of others, their positions, their skills, their
+   commitments, and their efforts.  We're respectful of the volunteer
+   efforts that permeate the Invenio community.  We're respectful of
+   the processes set forth in the community, and we work within them.
+   When we disagree, we are courteous in raising our issues.
+
+   Overall, we're good to each other.  We contribute to this community
+   not because we have to, but because we want to.  If we remember
+   that, these guidelines will come naturally.
```

Comment:
> While I like the spirit of your point, I'm somewhat concerned that we would be modifying Python's vanilla Code of Conduct instead of citing it literally.

Yep, but indeed I wanted to stress that we start from a somehow different community than the Python one and therefore we might need to acknowledge this into our code of conduct. Take it like an inheritance from Python code of conduct and where we override some of its methods :-) and make it more ours...